core/templates/dev/head/base.html

```diff
@@ -56,7 +56,7 @@
 
     <link rel="apple-touch-icon" href="/images/logo/favicon.png">
     <title itemprop="name">{% block subtitle %}{% endblock %} - {% block maintitle %}Oppia{% endblock %}</title>
-
+    <base href="/gallery">
```

Comment:
Oh, wait -- maybe this is because html5Mode was enabled? I guess what's confusing me is, go to the About page, and click a link in the sidebar. It doesn't work properly...